en/mkdocs.yml

```diff
@@ -56,7 +56,7 @@ nav:
     - 'Contribute': get-started/community.md
     - 'Feature Tour': get-started/quick-start-guide.md
   - 'Quick Starts':
-      - 'Overview': quick-starts/overview.md
+      #- 'Overview': quick-starts/overview.md
```

Comment:
Is this intentional?

Reply:
yes as we don't have an overview page for this section